capstone-frontend/src/components/BookSearchTile.jsx

```diff
@@ -1,14 +1,57 @@
 import React from 'react';
-import '../App.css';
+import { Container, Row, Col, DropdownButton, Dropdown } from 'react-bootstrap';
+import '../styles/BookSearchTile.css';
+
+const addBookToBookList = (bookId, bookListId) => {
+  console.log(`Added book with id: ${bookId} to bookList with id: ${bookListId}`);
+}
+
+const addBookToClub = (bookId, clubId) => {
+  console.log(`Added book with id: ${bookId} to club with id: ${clubId}`);
+}
+
+const addBookToCommunity = (bookId, communityId) => {
+  console.log(`Added book with id: ${bookId} to community with id: ${communityId}`);
+}
 
 const BookSearchTile = (props) => {
   return (
     <div className="book-search-tile">
-      <img className="book-img-med" src={props.thumbnailLink} alt={props.title}/>
-      <div>
-        <div className="book-title"> {props.title} </div>
-        <div className="book-author"> {props.author.join()} </div>
-      </div>
+      <Container>
+        <Row className="justify-content-md-center">
+          <Col md="auto"><img className="book-img-med" src={props.book.thumbnailLink} alt={props.book.title}/></Col>
+          <Col>
+            <div className="center-vertical">
+              <h2 className="book-title"> {props.book.title} </h2>
+              <p className="book-authors"> {props.book.authors.join(', ')} </p>
+            </div>
+          </Col>
+          <Col md="auto">
+            <Container className="center-vertical">
+            <DropdownButton id="dropdown-list-add" className="dropdown-add" title="Add to List">
+              {
+                props.userBookLists.map(bookList => 
+                  React.createElement(Dropdown.Item, { key:bookList.id,
```

Comment:
You should be able to use JSX here (e.g. `<Dropdown.Item key=... />` rather than `React.createElement`.

Reply:
Changed! I thought there must be a cleaner way to do it than using createElement